zend_abstract_interface/json/json.c

```diff
@@ -55,5 +135,140 @@ bool zai_json_setup_bindings(void) {
         php_json_serializable_ce = *tmp_json_serializable_ce;
     }
 
-    return zai_json_encode && zai_json_decode_ex;
+    return zai_json_encode != NULL;
```

Comment:
Not totally sure why we don't check for zai_json_decode_ex for PHP < 70100, or zai_json_parse & zai_json_parser_init for the other

Reply:
It's mostly - if one symbol is failing, the others will too, avoiding the extra check :-D.